packages/nimble-components/src/rich-text/mention-listbox/index.ts

```diff
@@ -287,6 +285,25 @@ export class RichTextMentionListbox extends FoundationListbox {
     private setOpen(value: boolean): void {
         this.open = value;
     }
+
+    // Aligns with select / combobox
+    // Modified to remove forced position concept
+    private setPositioning(options: MentionListboxShowOptions): void {
```

Comment:
Aligned to select setPositioning implementation without the forcedPositioning conceept

 https://github.com/ni/nimble/blob/aa21afb2ee252fa3333de6d0b209a7f54634d66b/packages/nimble-components/src/select/index.ts#L1064-L1089